app/src/views/private/private-view/components/private-view-sidebar.vue

```diff
@@ -0,0 +1,46 @@
+<script lang="ts" setup>
+import AiSidebarDetail from '@/ai/components/ai-sidebar-detail.vue';
+import { AccordionRoot } from 'reka-ui';
+import SkipMenu from '../../components/skip-menu.vue';
+import { useSidebarStore } from '../stores/sidebar';
+
+const sidebarStore = useSidebarStore();
+</script>
+
+<template>
+	<aside id="sidebar" ref="sidebarEl" role="contentinfo" class="alt-colors" aria-label="Module Sidebar">
```

Comment:
Let’s remove the ref attribute since it is unused.

```suggestion
	<aside id="sidebar" role="contentinfo" class="alt-colors" aria-label="Module Sidebar">
```